README.md

```diff
@@ -81,6 +76,8 @@ Note that `npm run update-local-config` will need to be re-run with some frequen
 
 1. Run `docker-compose build`.
 1. Run `docker-compose run app yarn` to install dependencies.
+1. Run `docker-compose run admin-client yarn` to install dependencies.
```

Comment:
There actually is a new `Makefile` that makes some of these steps clearer, maybe we should update the readme to point to it.

Reply:
That would be great but you may be the best person to document this option.